deis-output/.ruby-version

```diff
@@ -0,0 +1 @@
+2.2.2
```

Comment:
I guess this is why I like containerized build... nevermind my previous comment.  I actually _do_ have Ruby installed locally, but just because I used to be a Ruby guy.  But I _don't_ have 2.2.2. Containerizing would spare me some `rbenv` machincations.